cmd/clusterctl/client/cluster/mover.go

```diff
@@ -375,7 +375,7 @@ func patchCluster(proxy Proxy, cluster *node, patch client.Patch) error {
 	}
 
 	if err := cFrom.Patch(ctx, clusterObj, patch); err != nil {
-		return errors.Wrapf(err, "error pausing reconciliation for %q %s/%s",
+		return errors.Wrapf(err, "error patching %q %s/%s",
```

Comment:
Hope it's okay to just include this here (I found it during testing the inspect/logs code). The issue is that the `patchCluster` func is used to pause **and** unpause reconciliation so the previous message was sometimes misleading